### PR TITLE
hive/139751 | Page Hero

### PR DESCRIPTION
**Pull request by carloshww**
#### Short description:
Page Hero - 19 Hours
#### Jira ticket/Hive card: 
https://app.hive.com/workspace/LesfQJq3QdeFPsc8j?actionId=yqggfteRzQmDGsKs7
#### Changes proposed in this pull request:
- Component Hero

|Status|Statement|Value|
|----|----|----|
|:x:|Code was validated with the project linter|-|
|:x:|No broken or unnecessary images|-|
|:heavy_check_mark:|Development tested on Chrome|-|
|heavy_check_markx:|Development tested on Firefox|-|
|:heavy_check_mark:|Development tested on Safari|-|
|:x:|Development tested on IE11|-|
|:heavy_check_mark:|Responsive changes|-|
|:x:|CMS database changes|-|
|:x:|API Changes|-|
|:x:|Backend Changes|-|
|:heavy_check_mark:|Frontend Changes (Styles, JS, markup)|-|
|:x:|Copy Changes|-|
|:x:|Link Changes|-|
|:x:|301 Redirects|-|
|:x:|SEO Metatags|-|
|:heavy_check_mark:|Media updates|-|
|:x:|README file was updated with the relevant information of the project|-|

GITHUB: :heavy_check_mark: Accepted :x: Rejected :warning: Pending

#### Add screenshots:
image

image

### CODE REVIEW

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        }
+
+        &__desktop {
+            display: none;
+        }
+    }
+
+    &--copy {
+        position: absolute;
+        bottom: 2.063rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:27]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        text-align: center;
+        line-height: 1.2;
+        .intro {
+            font-size: .875rem;
+            margin-bottom: .313rem;
+        }
+        .title {
+            color: $color-white;
+            text-align: center;
+            margin: 0 1.875rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:39]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    &--copy {
+        position: absolute;
+        bottom: 2.063rem;
+        color: $color-white;
+        font-family: $font-barlow;
+        text-align: center;
+        line-height: 1.2;
+        .intro {
+            font-size: .875rem;
+            margin-bottom: .313rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:34]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+    &--copy {
+        position: absolute;
+        bottom: 2.063rem;
+        color: $color-white;
+        font-family: $font-barlow;
+        text-align: center;
+        line-height: 1.2;
+        .intro {
+            font-size: .875rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:33]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            margin: 0 1.875rem;
+            line-height: 1;
+            margin-bottom: .938rem;
+        }
+        .content {
+            &::before {
+                content: "";
+                display: block;
+                background: #c8f02d;
+                width: 4rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:48]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+                height: 0.375rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:49]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+                margin-top: 0;
+                border-radius: 0.938rem;
+                margin-bottom: 4px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:52]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+        }
+        .content {
+            &::before {
+                content: "";
+                display: block;
+                background: #c8f02d;
+                width: 4rem;
+                height: 0.375rem;
+                margin-top: 0;
+                border-radius: 0.938rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:51]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+                margin-bottom: 4px;
+                margin-left: auto;
+                margin-right: auto;
+            }
+            p {
+                font-size: 1.125rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:57]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+                font-family: $font-roboto;
+                margin-bottom: 2.188rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:59]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+        background-color: $color-las-palmas;
+        &--background {
+            &__mobile {
+                display: none;
+            }
+            &__desktop {
+                padding: 0;
+                display: block;
+                clip-path: polygon(0 0%, 100% 0, 100% 94%, 0% 100%);
+                margin-bottom: 1.25rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:84]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+                min-height: 43.125rem;
+                object-fit: cover;
+            }
+        }
+        &--copy {
+            max-width: 560px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:90]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+            top: 5.938rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:91]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 14]
```diff
@@ -0,0 +1,142 @@
...
+            left: 8.75rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:92]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+            text-align: left;
+            padding: 0;
+
+            .intro {
+                text-align: left;
+                font-size: 1.125rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:98]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+            }
+
+            .title {
+                margin: 0;
+                text-align: left;
+                font-size: 2.75rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:104]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+                line-height: 1.2;
+                span {
+                    font-size: 3.5rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:107]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+                }
+            }
+
+            .content {
+                display: flex;
+                flex-direction: column;
+                margin-left: 6.25rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:114]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+                padding: 0;
+                position: relative;
+
+                &::before {
+                    content: '';
+                    background: $color-las-palmas;
+                    width: 4rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:121]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+                    height: .375rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:122]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+                    margin-top: 2.688rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:123]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+                    border-radius: .938rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:124]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+                    margin-bottom: 1.875rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:125]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+                    margin-left: 0;
+                }
+
+                p {
+                    font-size: 1.375rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:130]: use rem function

File: WebSite/WebSite/Styles/components/_hero.scss [hunk 25]
```diff
@@ -0,0 +1,142 @@
...
+                    margin-bottom: 1.563rem;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_hero.scss:131]: use rem function

**Review by aldortsa — CHANGES_REQUESTED** [review]

File: WebSite/WebSite/wwwroot/img/home/Mask Group 87.png
Comment (aldortsa) [inline WebSite/WebSite/wwwroot/img/home/Mask Group 87.png:1]: home-hero-desktop.png

**Review by aldortsa — CHANGES_REQUESTED** [review]

File: WebSite/WebSite/Views/Shared/_Hero.cshtml [hunk 26]
```diff
@@ -1,20 +1,22 @@
...
+        <div class="content">
+            <p class="d-lg-none">
+                @Model.Mobile_content
+            </p>
+            <p class="d-none d-lg-block">
+                @Model.Mobile_content
+            </p>
+            @if (Model.ShowLink)
+            {
+                <a class="astellas-c-btn astellas-c-btn--primary" href="@Model.LinkAddress">@Model.LinkText</a>
```
Comment (randypasos) [inline WebSite/WebSite/Views/Shared/_Hero.cshtml:17]: Please use the _button.cshtml partial

**Review by randypasos — CHANGES_REQUESTED** [review]
Please address code comments, additionally, the desktop hero image is not being displayed. 
On mobile, the hero does not match the design. 
Desktop:
Screenshot 2023-04-19 at 17 56 01
Mobile:
Screenshot 2023-04-19 at 17 57 28

**Pull request merged**